website/reverse.php

```diff
@@ -29,7 +29,8 @@
 $fThreshold = $oParams->getFloat('polygon_threshold', 0.0);
 
 // Format for output
-$sOutputFormat = $oParams->getSet('format', array('html', 'xml', 'json', 'jsonv2'), 'xml');
+$sOutputFormat = $oParams->getSet('format', array('html', 'xml', 'json', 'jsonv2', 'geojson'), 'xml');
+$bAsGeoJSON = $bAsGeoJSON || $sOutputFormat === 'geojson';
```

Comment:
This always downloads the polygon geometry. Can you make it like the other formats, where you just get the centroid point, if a polygon parameter is not explicitly given?